frontend/src/app/modules/tenant-admin/client/client.component.spec.ts

```diff
@@ -1,10 +1,10 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 
 import { ClientComponent } from './client.component';
-import { RouterTestingModule } from '@angular/router/testing';
-import { ApiService } from '@services/api.service';
-import { SnackbarService } from '@services/snackbar.service';
-import { MatDialog } from '@angular/material/dialog';
+import {RouterTestingModule} from '@angular/router/testing';
+import {ApiService} from '@services/api.service';
+import {SnackbarService} from '@services/snackbar.service';
+import {MatDialog} from '@angular/material/dialog';
```

Comment:
Wir sollten vielleicht mal ne editorconfig einfügen oder klären, wie die Codeformatierung sein soll. Sonst gibt es ständig Umformatierungen, je nachdem, wer gerade arbeitet (aber out of scope für diesen PR)